### ISSUE TITLE
On Papers

### ISSUE THREAD

**Issue by Alex-Linhares** [Alex-Linhares/FARGonautica#2]
Ideally, anyone that copies this repo should have all papers... but what if the authors take issue with that?  Maybe we should have links instead?  

Anyways, for Bob French (need to consult him later):

http://leadserv.u-bourgogne.fr/files/publications/000291-subcognition-and-the-limits-of-the-turing-test.pdf

http://leadserv.u-bourgogne.fr/files/publications/000280-peeking-behind-the-screen-the-unsuspected-power-of-the-standard-turing-test.pdf

http://leadserv.u-bourgogne.fr/files/publications/000279-the-turing-test-the-first-50-years.pdf

http://leadserv.u-bourgogne.fr/files/publications/000273-the-computational-modeling-of-analogy-making.pdf

http://leadserv.u-bourgogne.fr/files/publications/000974-moving-beyond-the-turing-test.pdf

http://leadserv.u-bourgogne.fr/files/publications/000719-dusting-off-the-turing-test.pdf

http://leadserv.u-bourgogne.fr/files/publications/000763-computational-modeling-in-cognitive-science-a-manifesto-for-change.pdf

http://leadserv.u-bourgogne.fr/files/publications/001241-interactive-effects-of-explicit-emergent-structure-a-major-challenge-for-cognitive-computational-modeling.pdf

http://leadserv.u-bourgogne.fr/files/publications/001230-grandmother-cells-much-ado-about-nothing.pdf

**Comment by softwareloop** [comment]
Unfortunately "The Subtlety of Sameness" by Robert French seems out of print. Still, it's one of the best explanations of Tabletop and should be mentioned for those interested in finding it second hand. 
https://mitpress.mit.edu/books/subtlety-sameness

**Comment by eraoul** [comment]
I agree that the book is awesome. French’s thesis is similar but the book
is more clear, etc.

**Comment by Alex-Linhares** [comment]
I am trying to get the code for Tabletop, still waiting for a response from Bob.  

And YES!  The book is great and it's a pity that it's out of print.  I will mention it in the readme page; thanks a lot for the suggestion.  And also, of course, the text can benefit from review and pull-requests are very welcome!

**Comment by Paul-G2** [comment]
Just FYI, The Subtlety of Sameness book is available for online reading at the Internet Archive:
[https://archive.org/details/subtletyofsamene0000fren/page/n5/mode/2up](https://archive.org/details/subtletyofsamene0000fren/page/n5/mode/2up)

**Comment by Alex-Linhares** [comment]
> Just FYI, The Subtlety of Sameness book is available for online reading at the Internet Archive: https://archive.org/details/subtletyofsamene0000fren/page/n5/mode/2up

Thank you Paul; I've just included the link.
